Automattic/jetpack
# Admin Page: fix undefined Initial_State on npm run build and missing variable "text" in notices in dev mode

#### Changes proposed in this Pull Request:
- replace usage of Initial_State for property. This ensures it's defined and allows npm run build to complete correctly (previously it was throwing an undefined message and didn't allow to build)

- define `text` variable so it doesn't throw a warning in dev mode and allows execution to continue.
- group `if else` so it exits as soon as one matches

#### Testing instructions:
- do a `npm run build`, it should finish ok
- put the site in dev mode and visit Jetpack admin. It should display.
